pkg/profiles/profile.go

```diff
@@ -22,12 +23,27 @@ func GetStorageDirectory() string {
 	return filesystem.ConfigDir()
 }
 
+func GetDataDirectory() string {
+	p, err := Read()
+	if err != nil {
+		return filesystem.DataDir()
+	}
+	if p.CurrentExists() {
+		return *p.Current
+	}
+	return filesystem.DataDir()
+}
+
+func GetLogDirectory() string {
+	return filepath.Join(filesystem.DataDir(), "logs")
+}
+
 func FilePath() string {
 	return filepath.Join(filesystem.ConfigDir(), "profiles.json")
 }
 
-func Directories() string {
-	return filepath.Join(filesystem.ConfigDir(), "profiles")
+func Directories() (string, string) {
+	return filepath.Join(filesystem.ConfigDir(), "profiles"), filepath.Join(filesystem.DataDir(), "logs")
```

Comment:
@userappgate Where should the logs go according to XDG?
I have see the following mentioned as the "right" place: `$XDG_DATA_HOME`, `$XDG_STATE_HOME`, `$XDG_CACHE_HOME`.

https://superuser.com/questions/1293842/where-should-userspecific-application-log-files-be-stored-in-gnu-linux

Reply:
Apparently there is no "right" place, there doesn't seem to be any agreement on this.